config_samples/optimism/automaton/automaton_config_L1.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I think the configs shouldn't be located in the repo but automaton's repo. Contract addresses make no sense here